lib/AST/ASTImporter.cpp

```diff
@@ -4359,6 +4359,18 @@ ASTNodeImporter::VisitTemplateTemplateParmDecl(TemplateTemplateParmDecl *D) {
   return ToD;
 }
 
+// Returns the definition for a (forwad) declaration of a ClassTemplateDecl, if
+// it has any definition in the redecl chain.
+static ClassTemplateDecl* getDefinition(ClassTemplateDecl* D) {
```

Comment:
The asterisks are on the wrong side in the function prototype.

Reply:
I wonder if we should give a more specific name like `getClassTemplateDefinition`.